Weekly Update 2.md

```diff
@@ -0,0 +1,31 @@
+# Update for the 2nd week of self reports
+
+## 1.
+- This week I accomplished this weeks goals
```

Comment:
Good job on accomplishing this week's goals! Also, good job on creating S.M.A.R.T. goals. 
You can go ahead and merge this pull request.